cli/chaosli/cmd/create.go

```diff
@@ -83,6 +84,19 @@ func createSpec() (v1beta1.DisruptionSpec, error) {
 	spec.Selector = getSelectors()
 	spec.Count = getCount()
 
+	isPulsingCompatible := true
+
+	for _, disruptionKind := range spec.GetKindNames() {
+		if disruptionKind == types.DisruptionKindContainerFailure || disruptionKind == types.DisruptionKindNodeFailure {
```

Comment:
This validation is logically different from what we're performing in https://github.com/DataDog/chaos-controller/pull/473/files#diff-972800d1b1e32baefe4ea8e8b7e3cba2845eac77ce8f9ea9b026d168361d9181R229

Consider the following odd, but valid disruption spec:
```
spec:
  containerFailure:
  network:
    drop: 5
```

If we added a `pulse` field to this, the api would accept it, because we are using one of the pulsing compatible fields. But chaosli would not accept it, because we are using one of the pulsing incompatible fields as well. I'm fairly indifferent as to which direction you'd choose to support, but I do think we need to be consistent

Reply:
After some talk with Joris, we decided to just not allow the creation of disruptions with the pulse field if one of them is a container failure or a node failure. It also removes some of the complexity in the injector

Reply:
Yup to me it was very confusing that, in the given example, network disruption would be pulsing but not the container failure one because it is not compatible, and this without warning the user. Rejecting disruptions with pulsing on at least one incompatible failure ensures the user knows that it is not possible.